rust/tests/read_simple_table_test.rs

```diff
@@ -15,14 +15,15 @@ async fn read_simple_table() {
     assert_eq!(table.get_min_writer_version(), 2);
     assert_eq!(table.get_min_reader_version(), 1);
     assert_eq!(
-        table.get_files(),
+        table.get_files().sort(),
```

Comment:
sort returns `()`

Reply:
good catch, @akshay26031996 could you send a PR to correct the tests?

Reply:
Noted, let me go through this.